### PR TITLE
Installer

### PR DESCRIPTION
**Pull request by seeruk**
The backup command should come soon, I don't expect it to be tremendously difficult. This script should be updated at the same time as that is implemented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,9 +16,16 @@ does not require an active daemon to run.
 
 ## Installation
 
+The easiest method is with the installation script:
+
+```
+$ curl -s https://raw.githubusercontent.com/SeerUK/tid/master/install.sh | sh
```
Comment (seeruk) [inline README.md:22]: Bear in mind, if you want to try this command out, you'll need to swap `master` to `installer`.

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+	TID_OS=$(echo `uname`|tr '[:upper:]' '[:lower:]')
+
+	case "$TID_OS" in
+		# Minimalist GNU for Windows
+		mingw*) TID_OS='windows';;
+	esac
+}
+
+prepareTidDir() {
+    if test ! -d "$TID_DIR"; then
```
Comment (cj123) [inline install.sh]: i'm not sure if it's something to do with github or not, but the indentation of these ifs in these methods looks off?

**Review by cj123 — COMMENTED** [review]
just the one thing, other than that it all looks good.

File: install.sh (hunk 2 above)
Comment (seeruk) [reply install.sh]: Hmm, some of this indentation might be mixed!? Not sure how that's happened in IntelliJ...

**Review by cj123 — APPROVED** [review]

**Pull request merged**